src/common/obj_loader.h

```diff
@@ -73,12 +73,6 @@ std::unordered_map<std::string, pybind11::object> load_mesh_obj(const std::strin
             v(i, 1) = attrib.vertices[3*i+1];
             v(i, 2) = attrib.vertices[3*i+2];
 
-            if (attrib.normals.size() > 0) {
-                n(i, 0) = attrib.normals[3*i+0];
-                n(i, 1) = attrib.normals[3*i+1];
-                n(i, 2) = attrib.normals[3*i+2];
-            }
-
             if (attrib.texcoords.size() > 0) {
```

Comment:
Looks like this would fail for uv coords and colors as well -- can we fix this and add a small test with the file you uploaded?

Reply:
I don't think vertex colors will be a problem, since the OBJ file format forces every vertex to redefine the color, even if it's not unique. Unlike vertex normals and UV, which can be reused.

Reply:
Good point about colors. We should still make the same change for texcoords at the line below though